backend/test_step_time.py

```diff
@@ -1,15 +1,17 @@
 import unittest
 from Step_Time_Calculation import calculate_step_time
 from target_zone_estimation import estimate_target_zone
+import os
 
 THRESHOLD = 20.0 
+DATA_FILE_PATH = os.path.join(os.path.dirname(__file__), "tied belt OSS_f_1.tsv")
```

Comment:
Can we rename this file such that the name has no white space? Use `_` to replace whitespaces